src/data/global.json

```diff
@@ -59,6 +59,11 @@
                 }
             },
             "nav": [
```

Comment:
Navigation looks a little bit broken now on small desktop screens - right before we change to the burger menu. But no blocker for now

Reply:
Maybe we could change the naming to something shorter for now?